quiz-app/components/Quiz/index.js

```diff
@@ -1,25 +1,44 @@
+import { useState } from "react";
 import Question from "../Question";
 import Answer from "../Answer";
 import SubmitButton from "../SubmitButton";
-import { questionsArray } from "../../lib/quizData";
+import questionsArray from "../../lib/quizData";
+
+// key needs to be on the looped elements - the parent element
 
 export default function Quiz() {
+  //programatically generates an array of length questionsArray and fills with null values.
+  const [answers, setAnswers] = useState(
+    Array(questionsArray.length).fill(null)
+  );
+  const handleChange = (index, buttonType) => {
+    console.log({ index, buttonType });
+    setAnswers(answers.map((x, ind) => (ind === index ? buttonType : x)));
+  };
 
-  const [result, setResult] = useState();
+  console.log({ answers });
 
-    return (
-      <>
-        <Question text={ questionsArray[0] } />
-        <Answer />
-        <Question text={ questionsArray[1] } />
-        <Answer result={ setResult } />
-        <Question text={ questionsArray[2] } />
-        <Answer />
-        <SubmitButton />
-      </>
-    );
+  return (
+    <form>
+      <ol>
+        {questionsArray.map((questionText, index) => (
+          <li key={questionText}>
```

Comment:
Did you consider using uuid package (https://www.npmjs.com/package/uuid) for generating unique keys?